cpp/src/binaryop/compiled/binops_custom.cu

```diff
@@ -0,0 +1,195 @@
+extern template void dispatch_single_double<ops::GreaterEqual>(mutable_column_device_view&, column_device_view const&, column_device_view const&, rmm::cuda_stream_view);
+extern template void dispatch_single_double<ops::BitwiseAnd>(mutable_column_device_view&, column_device_view const&, column_device_view const&, rmm::cuda_stream_view);
+extern template void dispatch_single_double<ops::BitwiseOr>(mutable_column_device_view&, column_device_view const&, column_device_view const&, rmm::cuda_stream_view);
+extern template void dispatch_single_double<ops::BitwiseXor>(mutable_column_device_view&, column_device_view const&, column_device_view const&, rmm::cuda_stream_view);
+extern template void dispatch_single_double<ops::LogicalAnd>(mutable_column_device_view&, column_device_view const&, column_device_view const&, rmm::cuda_stream_view);
+extern template void dispatch_single_double<ops::LogicalOr>(mutable_column_device_view&, column_device_view const&, column_device_view const&, rmm::cuda_stream_view);
+extern template void dispatch_single_double<ops::ShiftLeft>(mutable_column_device_view&, column_device_view const&, column_device_view const&, rmm::cuda_stream_view);
+extern template void dispatch_single_double<ops::ShiftRight>(mutable_column_device_view&, column_device_view const&, column_device_view const&, rmm::cuda_stream_view);
+extern template void dispatch_single_double<ops::ShiftRightUnsigned>(mutable_column_device_view&, column_device_view const&, column_device_view const&, rmm::cuda_stream_view);
+extern template void dispatch_single_double<ops::LogBase>(mutable_column_device_view&, column_device_view const&, column_device_view const&, rmm::cuda_stream_view);
+extern template void dispatch_single_double<ops::ATan2>(mutable_column_device_view&, column_device_view const&, column_device_view const&, rmm::cuda_stream_view);
+extern template void dispatch_single_double<ops::PMod>(mutable_column_device_view&, column_device_view const&, column_device_view const&, rmm::cuda_stream_view);
+// extern template void dispatch_single_double<ops::NullEquals>(mutable_column_device_view&, column_device_view const&, column_device_view const&, rmm::cuda_stream_view);
+// extern template void dispatch_single_double<ops::NullMax>(mutable_column_device_view&, column_device_view const&, column_device_view const&, rmm::cuda_stream_view);
+// extern template void dispatch_single_double<ops::NullMin>(mutable_column_device_view&, column_device_view const&, column_device_view const&, rmm::cuda_stream_view);
+// extern template void dispatch_single_double<ops::UserDefinedOp>(mutable_column_device_view&, column_device_view const&, column_device_view const&, rmm::cuda_stream_view);
+// clang-format on
+
+void operator_dispatcher(mutable_column_view& out,
+                         column_view const& lhs,
+                         column_view const& rhs,
+                         binary_operator op,
+                         rmm::cuda_stream_view stream)
```

Comment:
Make this more generic by taking a callable to invoke with the dispatched operator non-type template parameter. Otherwise this logic is duplicated in `is_supported_operation_functor`.

Reply:
`is_supported_operation_functor` is modified now to reduce compile time of `util.cpp`.